faqs/asking_questions.md

```diff
@@ -0,0 +1,20 @@
+---
+title: How to Ask a Question
+search: ask question
+---
+Knowing how to effectively ask a question is essential in getting help from others. If you ask a vague question without many details, others have to slowly pry out what exactly you need. By stating everything up front, it's much easier to get answers because those who want to help don't have to ask a bunch of questions just to know what you want.
+
+**1. Give a summary of your problem**  
+State briefly what the issue you're running into is. Explain your goal and what problem is occurring. This should be no more than a few sentences; more detail will be added later.
+
+**2. Show the code or program you're running**  
+This is where you link to or embed the code you are using. If you're writing a program, paste in the code you wrote inside "\`\`\`lua" and "\`\`\`" brackets - this gives it some styling that makes it easier to read. (An example is listed below.) If you're using someone else's program, link the Pastebin or forum URL that you got it from.
```

Comment:
> An example is listed below.

I don't think this is still the case is it?

Reply:
Oop, fixed.